msmbuilder/decomposition/sparsetica.py

```diff
@@ -123,49 +111,35 @@ class SparseTICA(tICA):
     .. [3] Mackey, L. "Deflation Methods for Sparse PCA." NIPS. Vol. 21. 2008.
     """
 
-    def __init__(self, n_components, lag_time=1, gamma=0.05,
-                 rho=0.01, weighted_transform=True, epsilon=1e-6, tolerance=1e-8,
-                 maxiter=10000, max_nc=100, greedy=True, verbose=False):
-        super(SparseTICA, self).__init__(
-            n_components, lag_time=lag_time, gamma=gamma,
+    def __init__(self, n_components, lag_time=1, rho=0.01,
+                 weighted_transform=True, epsilon=1e-6, shrinkage=None,
+                 tolerance=1e-6, maxiter=10000, verbose=False):
```

Comment:
not having a default for n_components doesn't allow running SparseTICA() without arguments (which might be used e.g. in osprey). tICA can handle this and sets n_components=None by default and then later, if self.n_components is None: self.n_components = n_features